stablehlo/transforms/VhloLegalizeToStablehlo.cpp

```diff
@@ -202,6 +202,27 @@ SpecialResult specialSuccess() { return SpecialResult::SPECIAL_SUCCESS; }
 SpecialResult specialFailure() { return SpecialResult::SPECIAL_FAILURE; }
 SpecialResult notSpecial() { return SpecialResult::NOT_SPECIAL; }
 
+LogicalResult convertBool(Attribute vhloAttr, bool& result) {
```

Comment:
Should we create a `VhloUtils.cpp`? Some of this code is duplicated in other files.

Reply:
The functions are slightly different. `convertBool` here is StableHLO->VHLO, `convertBool`s other definition is VHLO->StableHLO

Reply:
There is some duplication between VHLO to version patterns this conversion though, but I think its a single 2-line function at this point so duplication is probably better than an early refactor.